products/sle15/profiles/pci-dss.profile

```diff
@@ -84,8 +85,12 @@ selections:
     - dconf_db_up_to_date
     - dconf_gnome_screensaver_lock_enabled
     - dconf_gnome_screensaver_idle_delay
+<<<<<<< HEAD
     - chronyd_or_ntpd_specify_multiple_servers
     - chronyd_specify_remote_server
+=======
+    - configure_opensc_card_drivers
+>>>>>>> cddcb61e6a... Add  configure_opensc_card_drivers, force_opensc_card_drivers and install_smartcard_packages rules to SLE15 PCI-DSS profile
```

Comment:
Please resolve this.

Reply:
Thanks :bow: 
Not sure how I missed it but handled the conflict and rebased the branch just in case.